doc/dadbod.txt

```diff
@@ -81,6 +81,33 @@ Handling these parameters is up to the individual adapters.  By default,
 unrecognized parameters are ignored.  Pass Boolean parameters as the literal
 strings "true" and "false" (e.g., `?ssl=true`).
 
+                                                *dadbod-bigquery*
+bigquery ~
+>
+    bigquery://[?<globalflag>=<value>][&<globalflag>=<value>][...]
+    bigquery://?format=sparse select count(*) from publicdata.samples.shakespeare
```

Comment:
Don't need any example of a query; none of the other adapters do that. 

Per my "the examples in the documentation should be limited to officially sanctioned connection parameters" remark, we don't want an example with `format=sparse` either.

Reply:
Removed 👍